src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,131 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        storageDao.add(new User("popova", "justpopova", 26));
+        storageDao.add(new User("sterniuk", "1123313", 30));
+        storageDao.add(new User("pony", "dfyrg6", 17));
+    }
+
+    @Test
+    public void userNull_isNotValid() {
+        Assertions.assertThrows(RuntimeException.class, () -> {
+            registrationService.register(null);
+        });
+    }
```

Comment:
```suggestion
    @Test
    public void register_userNull_isNotValid() {
        Assertions.assertThrows(RuntimeException.class, () -> {
            registrationService.register(null);
        });
    }
```
in this task it is obvious that this is register method, but there could be other method in same class, so we should specify particular method that we are testing

Reply:
clean your storage after each test case, so they are not interacting with each other